pages/intro.mdx

```diff
@@ -1,48 +1,19 @@
 # Introduction
 
-You’ve found the Discord Developer Documentation! These pages are dedicated to showing you all the ways that you can use Discord to make cool stuff. Whether you’re looking to create awesome bots for your community, empower your applications with our API, or hook us right into your game with Rich Presence or the GameSDK, Discord has something for you.
+You’ve found the Unofficial Discord User API Documentation! These pages are dedicated to showing you all the ways that you can use Discord to make cool stuff.
 
-All of our [documentation is on GitHub](https://github.com/discord/discord-api-docs) and we <3 corrections and improvements!
+All of our [documentation is on GitHub](https://github.com/discord-userdoccers/discord-userdoccers) and we <3 corrections and improvements!
 
-## Bugs
+## ⚠️ Note
 
-If you believe you're experiencing a bug with our API or want to report incorrect documentation, open an issue on our [issue tracker](https://github.com/discord/discord-api-docs/issues).
+This documentation is not affiliated with or endorsed by Discord in any way. It is a community effort to document the API as it is used by user accounts, and is not an official source of information. Automating user accounts is _against ToS_ (🙄 IKR), so just a heads up doing so unsafely might get you banned.
 
-## Bots and Apps
+## Scope
 
-Bots and apps are the lifeblood of the Discord development community. They come in all shapes and sizes, from small hobby projects for your server with friends, to huge projects that live in hundreds of thousands of servers. We love seeing the unique, fun, and sometimes downright strange (in a good way) creations that come from our community.
+Bots and interactions are not the focus of this documentation set, users are! If interactions are what you’re looking for head over to [Discord’s Official Documentation](https://discord.com/developers/docs/intro).
 
-Discord offers an open API to serve requests for both bots and OAuth2 integrations. So whether you’re making your own [`/wumpus` commands](/interactions/application-commands/) or looking to [`Log In With Discord`](/topics/oauth2/), we’ve got you covered.
+Discord offers an open API to serve requests for users, bots, and OAuth2 integrations. So whether you’re making your own `!dmall` command or looking to [`Aggregate Emails`](/topics/oauth2/), we’ve got you covered.
```

Comment:
Do we really need this?

Reply:
No but the intro page doesn't have much content

Reply:
True, maybe we could look at rewording this so as not to mention specific dodgy activities?

Reply:
Maybe 👽